config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -199,6 +199,42 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-windows
       description: "Run E2E tests on a Windows cluster for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-gcp
+    decorate: true
+    decoration_config:
+      timeout: 25m
+    always_run: true
+    optional: true
+    path_alias: sigs.k8s.io/secrets-store-csi-driver
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
```

Comment:
Please do not use Prow's infrastructure credentials. Your job specifies `decorate: true` so your test container shouldn't need credentials to upload to our GCS job results bucket itself. This makes it possible for your job to mistakenly overwrite job results.

Reply:
This SGTM for the GCP plugin, but this is set for all jobs, @aramase should I remove this label from the other jobs in this PR? I can also create a separate PR for that.

Reply:
@tam7t We can start by not having this label for GCP job. After we have that running in the driver and confirm the label isn't required we can open a separate PR to remove it from all the other jobs as well.

Reply:
SGTM Updated this PR to just remove it from the GCP job